react-events-and-state/src/index.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import ReactDOM from 'react-dom/client';
+
+class CustomButton extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleClick = this.handleClick.bind(this);
+    this.state = { isClicked: false };
+  }
+
+  UnclickButton(props) {
+    return <button onClick={this.handleClick}>{this.props.text}</button>;
+  }
+
+  ClickedButton(props) {
+    return <button> CLICKED! </button>;
+  }
+
+  handleClick(e) {
+    this.setState({ isClicked: true });
+  }
+
+  render(props) {
+    const isClicked = this.state.isClicked;
+    if (isClicked) {
+      return this.ClickedButton();
+    } else if (!isClicked) {
```

Comment:
You don't need an `else if` here, just an `else` will do the exact same thing. You never need to check the exact opposite of what was already checked. That is the point of an `else`.